msal/managed_identity.py

```diff
@@ -402,6 +432,8 @@ def _obtain_token(http_client, managed_identity, resource):
             os.environ["IDENTITY_HEADER"],
             os.environ["IDENTITY_SERVER_THUMBPRINT"],
             resource,
+            access_token_sha256_to_refresh=access_token_sha256_to_refresh,
```

Comment:
```suggestion
            token_sha256_to_refresh=access_token_sha256_to_refresh,
```

@rayluo according to the [spec](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/docs/msiv1_token_revocation.md) it is `token_sha256_to_refresh`

Reply:
TL;DR: This is not really a problem here, but still thanks for pointing this out.

Long explanation. There are (inevitably?) several internal layers of helpers. Currently, most of the helpers use a name "access_token_sha256_to_refresh", which "access" prefix was influenced by MSAL C++'s official parameter name "AccessTokenToRefresh". But we did use the right name "token_sha256_to_refresh" on the wire when sending out the request. You can confirm that by both the implementation inside `_obtain_token_on_service_fabric()` and by the test cases.

That being said, let me consider changing the existing internal variable names.

Reply:
agreed